internal/form/yes_no.go

```diff
@@ -4,6 +4,7 @@ package form
 type YesNo uint8
```

Comment:
with the explicit unknown it would be better to remove `-empty` and add `-trimprefix`